dags/rds_disaster_recovery/get_snapshots_dag.py

```diff
@@ -0,0 +1,282 @@
+import hashlib
+import json
```

Comment:
Can we copy the new changes from prod branch
https://github.com/NASA-IMPACT/csda-self-managed-airflow/blob/production/dags/rds_disaster_recovery/get_snapshots_dag.py